admin/src/Notifications/index.js

```diff
@@ -135,7 +135,7 @@ class Notifications extends React.Component {
 const notificationsWithFetch = withFetch(
 	`${HM.Workflows.Namespace}/notifications/${HM.Workflows.User}`,
 	{
-		expires:     10 * 1000,
+		expires:     60000,
```

Comment:
It might be easier to read in the old pattern, e.g.

```suggestion
		expires:     60 * 1000,
```

Reply:
Yeah true, not sure why I changed that.